nautobot/utilities/testing/views.py

```diff
@@ -278,7 +285,33 @@ def test_get_object_with_permission(self):
             obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
 
             # Try GET with model-level permission
-            self.assertHttpStatus(self.client.get(instance.get_absolute_url()), 200)
+            response = self.client.get(instance.get_absolute_url())
+            self.assertHttpStatus(response, 200)
+
+            # The object's display name or string representation should appear in the response
+            self.assertIn(getattr(instance, "display", str(instance)), str(response.content))
+
+            # If any Relationships are defined, they should appear in the response
+            if self.relationships:
+                for relationship in self.relationships:
+                    if ContentType.objects.get_for_model(instance) == relationship.source_type:
+                        self.assertIn(
+                            relationship.get_label(RelationshipSideChoices.SIDE_SOURCE), str(response.content)
+                        )
+                    if ContentType.objects.get_for_model(instance) == relationship.destination_type:
```

Comment:
Slight style/performance nit, to capture the CT in a variable since it is being evaluated twice each iteration.

```suggestion
                    content_type = ContentType.objects.get_for_model(instance)
                    if content_type == relationship.source_type:
                        self.assertIn(
                            relationship.get_label(RelationshipSideChoices.SIDE_SOURCE), str(response.content)
                        )
                    if content_type == relationship.destination_type:
```

Reply:
You caught me being lazy here. :-) Will do.